README.md

```diff
@@ -1,3 +1,97 @@
-# Heroku Buildpack: Blackfire Agent
+# Heroku Buildpack: Blackfire
 
-This is the official Heroku Buildpack for Blackfire Agent.
\ No newline at end of file
+> **If your app is written in PHP, you currently don't need to use this buildpack** as it is already [integrated in the
+> official PHP buildpack](https://blackfire.io/docs/integrations/paas/heroku).
+
+This is the official Heroku Buildpack for [Blackfire](https://blackfire.io).
+
+Blackfire is the Code Performance Management solution for developers
+to find and fix performance bottlenecks in dev, test/staging and production.
```

Comment:
Blackfire Profiler is a Software-as-a-Service tool which measures how your code consumes resources at run-time. It enables to find performance bottlenecks and understand the code's behavior. It can be used in development, test/staging and production (with no overhead for end-users).

Intuitive visualizations enable you to browse through your call stack, and review Wall-time, CPU time, I/O time, Memory, Network calls, HTTP requests and SQL queries usage.